corehq/motech/models.py

```diff
@@ -163,6 +168,79 @@ def last_token(self, token: Optional[dict]):
     def notify_addresses(self):
         return [addr for addr in re.split('[, ]+', self.notify_addresses_str) if addr]
 
+    @property
+    def is_throttled_connection(self):
+        return self.throttle_window is not None
+
+    def request_is_throttled(self):
+        return self._should_throttle_request()
```

Comment:
Any particular reason to not swap out the `_should_throttle_request` method name for `request_is_throttled` and throw away this method? All it does is call the `_should_throttle_request` method